packages/vats/src/ibc.js

```diff
@@ -624,8 +768,7 @@ export const prepareIBCProtocol = (zone, { makeVowKit, watch, when }) => {
               const conn = channelKeyToConnP.get(channelKey);
               const data = base64ToBytes(data64);
 
-              watch(conn.send(data), makeAckWatcher(util, packet));
-              break;
+              return watch(conn.send(data), makeAckWatcher(util, packet));
```

Comment:
This `makeAckWatcher` could be a singleton as well.
```suggestion
              return watch(conn.send(data), ackWatcher, { util, packet });
```